docs/api/main-config-preview-main-template.md

```diff
@@ -0,0 +1,105 @@
+---
+title: 'config.previewMainTemplate'
+---
+
+Type: `string`
+
+Default: See example below
+
+Override the preview's main page template, with a reference to a file containing an `.ejs` template that will be interpolated with environment variables.
```

Comment:
> interpolated with environment variables

This part felt incomplete without including the default template, but that's now a maintenance burden. Any concerns?